README.md

```diff
@@ -4,23 +4,25 @@
 
 ## Overview
 
-`agent-stack-k8s` is a Kubernetes [controller](https://kubernetes.io/docs/concepts/architecture/controller/) that uses the Buildkite [Agent API](https://buildkite.com/docs/apis/agent-api) to watch for scheduled jobs assigned to the controller's queue.
+The Buildkite Agent Stack for Kubernetes (also known as `agent-stack-k8s`) is a Kubernetes [controller](https://kubernetes.io/docs/concepts/architecture/controller/) that uses the Buildkite [Agent API](https://buildkite.com/docs/apis/agent-api) to watch for scheduled jobs assigned to the controller's queue.
 
 > [!NOTE] Starting with v0.28.0, the Buildkite GraphQL API is no longer used. If you are upgrading from an older version, your GraphQL-enabled token can be safely removed from your configuration or Kubernetes secret. Only the Agent token is required.
 
 ## Requirements
 
-- A Kubernetes cluster
-- A Cluster [Agent Token](https://buildkite.com/docs/agent/v3/tokens#create-a-token)
-- A Cluster [Queue](https://buildkite.com/docs/pipelines/clusters/manage-queues#create-a-self-hosted-queue)
-  - The UUID of this Cluster is also required. See [Obtain Cluster UUID](docs/installation.md#how-to-find-a-buildkite-clusters-uuid)
+- A Kubernetes cluster.
+- A [Buildkite cluster](https://buildkite.com/docs/pipelines/clusters/manage-clusters) and [agent token](https://buildkite.com/docs/agent/v3/tokens) for this cluster.
+- A [self-hosted queue](https://buildkite.com/docs/pipelines/clusters/manage-queues#create-a-self-hosted-queue) for this Buildkite cluster.
+  - The UUID of this Buildkite cluster is also required. Learn how to do this in [How to find a Buildkite cluster's UUID](https://buildkite.com/docs/agent/v3/agent-stack-k8s/installation#how-to-find-a-buildkite-clusters-uuid).
```

Comment:
I'm slightly in two minds about replicating this content in the README when it's now also mentioned on the Overview page of the Agent Stack for Kubernetes section of the Buildkite Docs.

While it adds convenience to see the requirements up front on this page, it also adds the overhead of having to maintain (now) 2 copies of this content - one here and the other one [on the Overview page of the official docs](http://buildkite.com/docs/agent/v3/agent-stack-k8s#before-you-start).

Anyway, we can see how this goes, but it'd imho, it's a crap reading experience when these two lists start going out of sync.